src/libotutil/ot-fs-utils.c

```diff
@@ -25,6 +25,54 @@
 #include <sys/xattr.h>
 #include <gio/gunixinputstream.h>
 
+/* Stub out old libglnx API to new one */
```

Comment:
Did that, also rebased on the new version of the API and cleaned up.  :arrow_down: